ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly-4.13-upgrade-from-stable-4.12.yaml

```diff
@@ -83,12 +74,28 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: aws-ipi-arm64-p2-f14
+  cron: 9 14 5,20 * *
```

Comment:
Add `cluster: arm01` to trigger the same job arm64 build cluster

Reply:
Thank you Deep!!